spec_3/index.js

```diff
@@ -0,0 +1,34 @@
+let example = "Catch me outside, how about that?"
+
+const dropEmpty = (array) => {
+  console.log(array)
+  let newArray = []
+  for(i = 0; i < array.length; i++){
+    let currentWord = array[i]
+    if( currentWord){
+      newArray.push(currentWord)
+    }
+  }
+  console.log('new',newArray)
+  return newArray
+}
+
+const stringIndex = (index, phrase) => {
+  let phraseAsArray = splitOnSpace(phrase)
+  console.log(phraseAsArray)
+  phraseAsArray = dropEmpty(phraseAsArray)
+  return phraseAsArray[index - 1]
+}
+
+const splitOnSpace = (word) => {
+  const wordArray = word.split(' ')
```

Comment:
This function again presents the opportunity for a more concise version by chaining the operations:
```js
const splitOnSpace = word =>
  word.split( ' ' ).map( stripPunctuation )
```